src/ApiService/ApiService/OneFuzzTypes/Requests.cs

```diff
@@ -81,8 +84,15 @@ public record NodeStateUpdate(
 [JsonConverter(typeof(SubclassConverter<NodeStateData>))]
 public abstract record NodeStateData;
 
+public record NodeSettingUpData(
+    [property: Required] Guid JobId,
```

Comment:
There's a small chance of failure here if:

1. Node runs setup script and pulls an 'old' agent binary
2. onefuz service updates
3. onefuzz node emits old setting up event

We can always resubmit so it's not a huge deal, just we should be aware of it when deploying.